test/mobile/features/steps/unity_steps.rb

```diff
@@ -0,0 +1,32 @@
+When("I wait for the game to start") do
+  # Wait for a fixed time period
+  sleep 10
+end
+
+When("I tap the {string} button") do |button|
+  # TODO Currently specific to the ANDROID_9_0 device (Google Pixel 3)
```

Comment:
I did a little bit of research and think the Unity approach for supporting other Android devices by using [Canvas with screen space overlay enabled](https://docs.unity3d.com/Packages/com.unity.ugui@1.0/manual/UICanvas.html#screen-space---overlay) to draw the buttons (which I think we already do), and [Screen](https://docs.unity3d.com/ScriptReference/Screen-height.html) to get the width/height/dpi. 

The Appium approach (which might be better) looks like it would involve getting the [window size](https://appium.io/docs/en/commands/web/window/get-window-size/) - and then we could try and alter the UI in Unity so that it's always a fixed percentage of the screen.

Reply:
There's certainly a lot of scope for looking at how we take this forward and it's likely to be a team effort on the approach.